[
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/257347275",
    "html_url": "https://github.com/electron/electron/issues/7792#issuecomment-257347275",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/7792",
    "id": 257347275,
    "user": {
      "login": "kevinsawicki",
      "id": 671378,
      "avatar_url": "https://avatars0.githubusercontent.com/u/671378?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/kevinsawicki",
      "html_url": "https://github.com/kevinsawicki",
      "followers_url": "https://api.github.com/users/kevinsawicki/followers",
      "following_url": "https://api.github.com/users/kevinsawicki/following{/other_user}",
      "gists_url": "https://api.github.com/users/kevinsawicki/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/kevinsawicki/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/kevinsawicki/subscriptions",
      "organizations_url": "https://api.github.com/users/kevinsawicki/orgs",
      "repos_url": "https://api.github.com/users/kevinsawicki/repos",
      "events_url": "https://api.github.com/users/kevinsawicki/events{/privacy}",
      "received_events_url": "https://api.github.com/users/kevinsawicki/received_events",
      "type": "User",
      "site_admin": true
    },
    "created_at": "2016-10-31T16:42:22Z",
    "updated_at": "2016-10-31T16:42:22Z",
    "body": "> Electron version: 1.3.6\n\nAre you able to reproduce this on 1.4.4 as well? Are there any errors logged to the `Console.app` when this happens?\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/257446858",
    "html_url": "https://github.com/electron/electron/issues/7792#issuecomment-257446858",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/7792",
    "id": 257446858,
    "user": {
      "login": "fancyydk",
      "id": 12807659,
      "avatar_url": "https://avatars0.githubusercontent.com/u/12807659?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/fancyydk",
      "html_url": "https://github.com/fancyydk",
      "followers_url": "https://api.github.com/users/fancyydk/followers",
      "following_url": "https://api.github.com/users/fancyydk/following{/other_user}",
      "gists_url": "https://api.github.com/users/fancyydk/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/fancyydk/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/fancyydk/subscriptions",
      "organizations_url": "https://api.github.com/users/fancyydk/orgs",
      "repos_url": "https://api.github.com/users/fancyydk/repos",
      "events_url": "https://api.github.com/users/fancyydk/events{/privacy}",
      "received_events_url": "https://api.github.com/users/fancyydk/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-10-31T23:10:02Z",
    "updated_at": "2016-10-31T23:10:02Z",
    "body": "@kevinsawicki, thanks for the quick reply.\n\nUnfortunately, I myself cannot reproduce this error. I saw it in the logs from a couple of users and it looks like the autoUpdater sucessfully downloaded the file but that's all I can tell for now. They are all on version 1.3.6.\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/258404151",
    "html_url": "https://github.com/electron/electron/issues/7792#issuecomment-258404151",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/7792",
    "id": 258404151,
    "user": {
      "login": "salomvary",
      "id": 193923,
      "avatar_url": "https://avatars2.githubusercontent.com/u/193923?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/salomvary",
      "html_url": "https://github.com/salomvary",
      "followers_url": "https://api.github.com/users/salomvary/followers",
      "following_url": "https://api.github.com/users/salomvary/following{/other_user}",
      "gists_url": "https://api.github.com/users/salomvary/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/salomvary/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/salomvary/subscriptions",
      "organizations_url": "https://api.github.com/users/salomvary/orgs",
      "repos_url": "https://api.github.com/users/salomvary/repos",
      "events_url": "https://api.github.com/users/salomvary/events{/privacy}",
      "received_events_url": "https://api.github.com/users/salomvary/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-11-04T11:05:42Z",
    "updated_at": "2016-11-04T11:05:42Z",
    "body": "I saw this happening when called `autoUpdater.checkForUpdates()` while it was in the middle of checking for updates. Simplest way to reproduce:\n\n``` js\nautoUpdater.checkForUpdates()\nautoUpdater.checkForUpdates()\n```\n\nPossible fixes on Electron:\n- `autoUpdater.checkForUpdates()` is a no-op if it is already checking\n- Expose a `autoUpdater.status` with meaningful value on the progress\n\nPossible fix (or workaround) outside of Electron:\n- Listen to autoUpdater events and maintain the state yourself. Do not do `autoUpdater.checkForUpdates()` if it is already happening.\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/302240080",
    "html_url": "https://github.com/electron/electron/issues/7792#issuecomment-302240080",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/7792",
    "id": 302240080,
    "user": {
      "login": "justechn",
      "id": 1508662,
      "avatar_url": "https://avatars2.githubusercontent.com/u/1508662?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/justechn",
      "html_url": "https://github.com/justechn",
      "followers_url": "https://api.github.com/users/justechn/followers",
      "following_url": "https://api.github.com/users/justechn/following{/other_user}",
      "gists_url": "https://api.github.com/users/justechn/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/justechn/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/justechn/subscriptions",
      "organizations_url": "https://api.github.com/users/justechn/orgs",
      "repos_url": "https://api.github.com/users/justechn/repos",
      "events_url": "https://api.github.com/users/justechn/events{/privacy}",
      "received_events_url": "https://api.github.com/users/justechn/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-05-17T21:42:42Z",
    "updated_at": "2017-05-17T21:42:42Z",
    "body": "I am also running into this and I am on electron@1.6.2"
  }
]